content/references/protocol-reference/transactions/transaction-types/accountset.md

```diff
@@ -74,15 +74,15 @@ The available AccountSet flags are:
 | `asfDefaultRipple`                | 8             | `lsfDefaultRipple`                | Enable [rippling](rippling.html) on this account's trust lines by default. |
 | `asfDepositAuth`                  | 9             | `lsfDepositAuth`                  | Enable [Deposit Authorization](depositauth.html) on this account. _(Added by the [DepositAuth amendment][].)_ |
 | `asfDisableMaster`                | 4             | `lsfDisableMaster`                | Disallow use of the master key pair. Can only be enabled if the account has configured another way to sign transactions, such as a [Regular Key](cryptographic-keys.html) or a [Signer List](multi-signing.html). |
-| `asfDisallowIncomingCheck`        | 13            | `lsfDisallowIncomingCheck`        | Block incoming checks to the account; enforced by `rippled`. _(Added by the [DisallowIncoming amendment][].)_ |
-| `asfDisallowIncomingNFTokenOffer` | 12            | `lsfDisallowIncomingNFTokenOffer` | Block incoming NFToken offers to the account; enforced by `rippled`. _(Added by the [DisallowIncoming amendment][].)_ |
-| `asfDisallowIncomingPayChan`      | 14            | `lsfDisallowIncomingPayChan`      | Block incoming payment channels to the account; enforced by `rippled`. _(Added by the [DisallowIncoming amendment][].)_ |
-| `asfDisallowIncomingTrustline`    | 15            | `lsfDisallowIncomingTrustline`    | Block incoming trust lines to the account; enforced by `rippled`. _(Added by the [DisallowIncoming amendment][].)_ |
-| `asfDisallowXRP`                  | 3             | `lsfDisallowXRP`                  | XRP should not be sent to this account. (Enforced by client applications, not by `rippled`) |
+| `asfDisallowIncomingCheck`        | 13            | `lsfDisallowIncomingCheck`        | Block incoming Checks. _(Requires the [DisallowIncoming amendment][] :not_enabled:)_ |
+| `asfDisallowIncomingNFTokenOffer` | 12            | `lsfDisallowIncomingNFTokenOffer` | Block incoming NFTokenOffers. _(Requires the [DisallowIncoming amendment][] :not_enabled:)_ |
+| `asfDisallowIncomingPayChan`      | 14            | `lsfDisallowIncomingPayChan`      | Block incoming Payment Channels. _(Requires the [DisallowIncoming amendment][] :not_enabled:)_ |
+| `asfDisallowIncomingTrustline`    | 15            | `lsfDisallowIncomingTrustline`    | Block incoming trust lines. _(Requires the [DisallowIncoming amendment][] :not_enabled:)_ |
+| `asfDisallowXRP`                  | 3             | `lsfDisallowXRP`                  | XRP should not be sent to this account. (Advisory; not enforced by the XRP Ledger protocol.) |
```

Comment:
We would like to reduce mentions of the `rippled` name since there have been some credible motions towards renaming the executable `xrpld` or something. The "enforced by `rippled` text seems more distracting than necessary here, since that _should_ be the expectation for all flag behavior unless stated otherwise.

I think the previous wording was based on the `asfDisallowXRP` flag setting a poor example, so I reworded it.